packages/app/vite.config.ts

```diff
@@ -1,9 +1,10 @@
 import { defineConfig } from 'vite'
 import mkcert from 'vite-plugin-mkcert'
 import solidPlugin from 'vite-plugin-solid'
+import solidSvg from 'vite-plugin-solid-svg'
 
 export default defineConfig({
-  plugins: [solidPlugin(), mkcert()],
+  plugins: [solidPlugin(), solidSvg({ defaultAsComponent: true }), mkcert()],
```

Comment:
This mkcert has issues on stackblitz, perhaps somehow having a switch to disable this easily, or removing for now completely can be considered